Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM golang:1.15.2-buster as builder
+WORKDIR /statsd/
+COPY . .
+RUN CGO_ENABLED=0 go build -ldflags '-extldflags "-static"' -o statsdaemon
```

Comment:
I'm not familiar with `-ldflags '-extldflags "-static"'` and I don't think it's necessary, but it doesn't seem to hurt.

Reply:
(I think `CGO_ENABLED=0` does the trick for pure-go programs like statsdaemon ... the `-extldflags -static` is just for programs that need cgo enabled to link to c libraries, to do that build and link statically, I think.)

Reply:
yeah that's a good point, those linker flags are just something I generally include to cover my bases but you're right in pointing out that they are not required here.

Reply:
👍